beacon-chain/rpc/validator_server_test.go

```diff
@@ -887,87 +890,88 @@ func TestMultipleValidatorStatus_OK(t *testing.T) {
 	}
 }
 
-func TestFilterActivePublicKeys(t *testing.T) {
-	currentEpoch := uint64(15)
-	beaconState := &pbp2p.BeaconState{
-		Slot: helpers.StartSlot(currentEpoch),
-		ValidatorRegistry: []*pbp2p.Validator{
-			// Active validiators in our request
-			{
-				Pubkey:          []byte("pk1"),
-				ActivationEpoch: currentEpoch - 1,
-				ExitEpoch:       math.MaxUint64,
-			},
-			// Inactive validators in our request
-			{
-				Pubkey:          []byte("pk2"),
-				ActivationEpoch: currentEpoch - 2,
-				ExitEpoch:       currentEpoch - 1,
-			},
-			// Other active validators in the registry
-			{
-				Pubkey:          []byte("pk3"),
-				ActivationEpoch: 0,
-				ExitEpoch:       math.MaxUint64,
-			},
-		},
-	}
-
-	vs := &ValidatorServer{}
+func BenchmarkAssignment(b *testing.B) {
```

Comment:
@terencechain I would call b.StopTimer here so all these other peripheral operations are not accounted for until necessary